### PR TITLE
Implemented Clanging Scales' effect

### PR DESCRIPTION
**Pull request by LOuroboros**
## Description
Addresses #1485 
Untitled5111

## **Discord contact info**
Lunos#4026

### CODE REVIEW

File: src/data/battle_moves.h [hunk 1]
```diff
@@ -10044,7 +10044,7 @@ const struct BattleMove gBattleMoves[MOVES_COUNT] =
 
     [MOVE_CLANGING_SCALES] =
     {
-        .effect = EFFECT_DEFENSE_DOWN_HIT,
+        .effect = EFFECT_ATTACKER_DEFENSE_DOWN_HIT,
         .power = 110,
         .type = TYPE_DRAGON,
         .accuracy = 100,
```
Comment (ultima-soul) [inline src/data/battle_moves.h:10050]: Also change `secondaryEffectChance` to equal 0 since 100% chance is handled by the battle script.

**Review by ultima-soul — CHANGES_REQUESTED** [review]

**Comment by LOuroboros** [comment]
Ready for re-review :eyes:

**Comment by ultima-soul** [comment]
Please resolve conflicts by merging the latest commits.

**Comment by LOuroboros** [comment]
> 
> 
> Please resolve conflicts by merging the latest commits.

Done :eyes:

**Review by ultima-soul — APPROVED** [review]

**Pull request merged**